.ado/templates/npm-publish.yml

```diff
@@ -0,0 +1,26 @@
+parameters:
+  # If this is a new stable branch, change `publishTag` to `latest` when going stable
+  publishTag: 'next'
```

Comment:
Right, this is something I wonder if we can check in CI and fast fail if it's wrong. Maybe a follow up

Reply:
This only needs to be changed once when going from RC to stable. In `getTagForStableBranch()`, it is only used to determine whether a branch is still in RC (after we check whether the current branch is greater than latest):

https://github.com/microsoft/react-native-macos/blob/2a1da0e249275b7afdbe5f82dad12082ccdec87a/.ado/scripts/prepublish-check.mjs#L116-L122

Reply:
I've added a quick check in the prepublish script.